models-info/src/main/java/com/flipkart/fdp/ml/transformer/VectorBinarizerTranformer.java

```diff
@@ -0,0 +1,55 @@
+package com.flipkart.fdp.ml.transformer;
+
+import com.flipkart.fdp.ml.modelinfo.VectorBinarizerModelInfo;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by karan.verma on 09/11/16.
+ */
+
+
+public class VectorBinarizerTranformer implements Transformer {
+    private final VectorBinarizerModelInfo modelInfo;
+
+    public VectorBinarizerTranformer(final VectorBinarizerModelInfo modelInfo) {
+        this.modelInfo = modelInfo;
+    }
+
+    @Override
+    public void transform(Map<String, Object> input) {
+        Object value = input.get(modelInfo.getInputKeys().iterator().next());
+        double[] inp = (value == null)? null: (double[])value;
```

Comment:
Not sure if this is any different from this:
double[] inp = (double[])value;
Please verify